templates/shared_templates/_helpers.tpl

```diff
@@ -84,6 +84,40 @@ storageClassName: {{ .Values.resources.requests.storage.classes.diskColumns | qu
 {{ end }}
 {{- end }}
 
+{{ define "rondb.ndbmtd.storageSize" -}}
+{{- $ := .root }}
+{{- if $.Release.IsInstall }}
+{{- $requiredStorage := add 
+    (div $.Values.resources.limits.memory.ndbmtdsMiB 1024 | int)
```

Comment:
I believe you meant the storage attribute, otherwise the size is still dependent of the memory